src/utility/HCISharedMemTransport.cpp

```diff
@@ -725,6 +817,41 @@ int HCISharedMemTransportClass::bt_ipm_set_addr(void)
   return 0; /* Error */
 }
 
+int HCISharedMemTransportClass::bt_ipm_set_random_addr(void)
+{
+  /* the specific table for set addr is 8 bytes:
+   * one byte for config_offset
+   * one byte for length
+   * 6 bytes for payload */
+  uint8_t data[4 + 8];
```

Comment:
If 8 bytes why +4 ?

Reply:
Yes Frederic, the comment refers only to payload as for the other messages implemented by Francois. I have just kept the structure of the code used by Francois in the other messages.